static/app/utils/shouldPreloadData.tsx

```diff
@@ -0,0 +1,56 @@
+import type {Config} from 'sentry/types/system';
+
+function getPathParams(path: string):
```

Comment:
Do we need all this logic? Couldn't we just check if the pathname starts with `/accept/` and in that case set shouldPreload to false?

Reply:
We could, but typically we perform a full pathname check, and I thought it was best to stick with that. However, on second thought, I believe it won’t cause any issues. Let me simplify it 🙂